SandboxTest/Main.cpp

```diff
@@ -227,6 +227,12 @@ int wmain (int argc, wchar_t *argv[]) {
         CHECK(loader->GetImageSource(&source));
     }
 
+    {
+        CComBSTR sopInstanceUID;
+        CHECK(source->GetSopInstanceUID(&sopInstanceUID));
+        std::wcout << "SOP Instance UID: " << sopInstanceUID.m_str << L"\n";
```

Comment:
One small comment: Can you convert the "SOP Instance UID: " text string to wide-char, so that it matches "wcout"?

Reply:
Good catch.

Reply:
Done. Also fixed missed `L` near `"Probe name"`